src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="field">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form_field">
```

Comment:
I recommended you put 1 empty line between open and closed tags like that
It would be more readable
```suggestion
        </legend>
        
        <label class="form_field">
```